From: Tom Herbert <therbert@google.com>
Date: Wed, 5 May 2010 14:03:32 +0000
Subject: e1000e: Save irq into netdev structure
References: bnc#699092, fate#311825
Patch-mainline: v2.6.35
Git-commit: f85e4dfac666e41b91e2b77fa563398e9379d5eb

Set net->devirq to pdev->irq.  This should be consistent with other
drivers.

Signed-off-by: Tom Herbert <therbert@google.com>
Acked-by: Bruce Allan <bruce.w.allan@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/e1000e/netdev.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/e1000e/netdev.c
+++ b/drivers/net/e1000e/netdev.c
@@ -5362,6 +5362,8 @@ static int __devinit e1000_probe(struct
 
 	SET_NETDEV_DEV(netdev, &pdev->dev);
 
+	netdev->irq = pdev->irq;
+
 	pci_set_drvdata(pdev, netdev);
 	adapter = netdev_priv(netdev);
 	hw = &adapter->hw;
